src/components/App.jsx

```diff
@@ -92,10 +101,11 @@ const Grid = styled.div`
 const Main = styled.main`
   display: grid;
   justify-content: center;
-  grid-template-rows: 170px min-content auto;
+  grid-template-rows: 5em min-content auto;
   grid-template-columns: 1fr;
   padding: clamp(30px, 10%, 100px) clamp(15px, 5%, 50px);
   overflow-y: auto;
+  overflow-x: hidden;
 `
```

Comment:
These two declaration could be merged into `overflow: hidden auto;`

Reply:
💡 done!